.github/workflows/main.yml

```
@@ -0,0 +1,154 @@
+
+  # Prepare release draft (if applicable)
+  prepare-release:
+    name: Prepare release
+    needs: vars
+    if: ${{ needs.vars.outputs.is_release }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+      - id: release-draft
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)